featuretools/computational_backends/feature_set_calculator.py

```diff
@@ -339,53 +340,53 @@ def _calculate_features(self, df, df_trie, features, progress_callback):
+        Merge ancestor_relationship_columns from parent_df into child_df, adding a prefix to
         each column name specifying the relationship.
 
-        Return the updated df and the new relationship variable names.
+        Return the updated df and the new relationship column names.
 
         Args:
-            child_df (pd.DataFrame): The dataframe to add relationship variables to.
-            parent_df (pd.DataFrame): The dataframe to copy relationship variables from.
-            ancestor_relationship_variables (list[str]): The names of
-                relationship variables in the parent_df to copy into child_df.
+            child_df (pd.DataFrame): The dataframe to add relationship columns to.
+            parent_df (pd.DataFrame): The dataframe to copy relationship columns from.
+            ancestor_relationship_columns (list[str]): The names of
+                relationship columns in the parent_df to copy into child_df.
             relationship (Relationship): the relationship through which the
                 child is connected to the parent.
         """
         relationship_name = relationship.parent_name
-        new_relationship_variables = ['%s.%s' % (relationship_name, var)
-                                      for var in ancestor_relationship_variables]
+        new_relationship_columns = ['%s.%s' % (relationship_name, var)
+                                    for var in ancestor_relationship_columns]
 
         # create an intermediate dataframe which shares a column
         # with the child dataframe and has a column with the
         # original parent's id.
-        col_map = {relationship.parent_column.id: relationship.child_column.id}
-        for child_var, parent_var in zip(new_relationship_variables, ancestor_relationship_variables):
+        col_map = {relationship.parent_column.name: relationship.child_column.name}
+        for child_var, parent_var in zip(new_relationship_columns, ancestor_relationship_columns):
             col_map[parent_var] = child_var
 
         merge_df = parent_df[list(col_map.keys())].rename(columns=col_map)
 
         merge_df.index.name = None  # change index name for merge
 
-        # Merge the dataframe, adding the relationship variables to the child.
+        # Merge the dataframe, adding the relationship columns to the child.
         # Left outer join so that all rows in child are kept (if it contains
-        # all rows of the entity then there may not be corresponding rows in the
+        # all rows of the dataframe then there may not be corresponding rows in the
         # parent_df).
         df = child_df.merge(merge_df,
                             how='left',
-                            left_on=relationship.child_column.id,
-                            right_on=relationship.child_column.id)
+                            left_on=relationship.child_column.name,
+                            right_on=relationship.child_column.name)
```

Comment:
use `relationship._child_column_name`

Reply:
Update this and all the other uses of this I could find.